src/hmrc-frontend-print-overrides.scss

```diff
@@ -0,0 +1,13 @@
+@import "components/all";
+@import "../../govuk-frontend/govuk/all";
+
+.govuk-back-link,
+.govuk-summary-list__actions,
+.govuk-footer__section,
+.govuk-footer__inline-list,
+.hmrc-report-technical-issue,
+.hmrc-language-select {
+  @include govuk-media-query($media-type: print) {
+    display: none !important;
+  }
+}
```

Comment:
I think it might be worth removing the imports and just having a plain
```
@media print {
  govuk-back-link,
  .govuk-summary-list__actions,
  .govuk-footer__section,
  .govuk-footer__inline-list,
  .hmrc-report-technical-issue,
  .hmrc-language-select {
    display: none !important;
  }
}
```

Reply:
(context for why they have the govuk mixin - I think - is that it lets you target a breakpoint by name - so you can say "desktop" and it has a k/v map that translates that into the viewport dimensions you need to apply styles at to cover that alias - but if we're just targeting by media then it's not doing more than a css `@media {}`)